environments/dotnet/FissionContext.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+
+namespace Fission.DotNetCore.Api
+{
+    public class FissionContext
+    {
+        public FissionContext(Dictionary<string, object> args, Logger logger)
+        {
+            if (args == null) throw new ArgumentNullException(nameof(args));
+            if (logger == null) throw new ArgumentNullException(nameof(logger));
+            Arguments = args;
+            Logger = logger;
+        }
+
+        public Dictionary<string, object> Arguments { get; private set; }
+        public Logger Logger { get; private set; }
+    }
+
+    public class Logger
```

Comment:
I've built a lightweight single-file logging implementation for C# called the Logary Facade. https://github.com/logary/logary#using-logary-in-a-library (scroll down for the C# example) You can then position an arbitrary logging framework behind the facade, should you wish to (like Logary itself)